files/collaborative_document_in_person.md

```diff
@@ -0,0 +1,88 @@
+![](https://i.imgur.com/iywjz8s.png)
+
+
+# Collaborative Document
+
+YYYY-MM-DD Workshop Type.
+
+Welcome to The Workshop Collaborative Document.
+
+This Document is synchronized as you type, so that everyone viewing this page sees the same text. This allows you to collaborate seamlessly on documents.
+
+----------------------------------------------------------------------------
+
+This is the Document for today: [link](<url>)
+
+Collaborative Document day 1: [link](<url>)
+
+Collaborative Document day 2: [link](<url>)
+
+Collaborative Document day 3: [link](<url>)
+
+Collaborative Document day 4: [link](<url>) 
+
+## 👮Code of Conduct
+
+Participants are expected to follow these guidelines:
+* Use welcoming and inclusive language.
+* Be respectful of different viewpoints and experiences.
+* Gracefully accept constructive criticism.
+* Focus on what is best for the community.
+* Show courtesy and respect towards other community members.
+ 
+## ⚖️ License
+
+All content is publicly available under the Creative Commons Attribution License: [creativecommons.org/licenses/by/4.0/](https://creativecommons.org/licenses/by/4.0/).
+
+## 🙋Getting help
+
+To ask a question, just raise your hand.
+
+If you need help from a helper, place a red sticky note on your laptop. A helper will come to assist you as soon as possible.
+
+## 🖥 Workshop website
+
+[link](<url>)
+
+🛠 Setup
+
+[link](<url>)
+
+Download files
+
+[link](<url>)
+
+## 👩‍🏫👩‍💻🎓 Instructors
+
+Instructor 1, Instructor 2
+
+## 🧑‍🙋 Helpers
+
+Helper 1, Helper 2  
+
+## 👩‍💻👩‍💼👨‍🔬🧑‍🔬🧑‍🚀🧙‍♂️🔧 Roll Call
+Name/ pronouns (optional) / job, role / social media (twitter, github, ...) / background or interests (optional) / city
+
+## 🗓️ Agenda
+| Time | Topic |
+|--:|:---|
+| 9:00 | 	Welcome and icebreaker |
+| 9:15 |  |
+| 10:15 | Coffee break |
+| 10:30 | |
+|  | _etc_ |
+
+## 🏢 Location logistics
+* Coffee and toilets are in the hallway, just outside of the classroom.
+* If you leave the building, 
+  be sure to be accompanied by someone from the escience center to let you back in through the groundfloor door
+* For access to this floor you might need to ring the doorbell so someone can let you in
+* In case of an emergency, you can exit our floor using the main staircase.
+  Or use the emergency staircase that is on the northside of the building in the hallway.
+* **Wifi**: Eduroam should work. Otherwise use the 'matrixbuilding' network, see A4 papers on the walls.
```

Comment:
```suggestion
* **Wifi**: Eduroam should work. Otherwise use the 'matrixbuilding' network, password should be printed out and available somewhere in the room.
```